src/ipyforcegraph/behaviors/_base.py

```diff
@@ -21,6 +21,9 @@ class Behavior(ForceBase):
 
     _model_name: str = T.Unicode("BehaviorModel").tag(sync=True)
 
+    CONTEXT: Optional[str] = None
+    RANK: Optional[int] = None
```

Comment:
i think this would be a trait, with sensible `T.default` values per class, that could be overridden when invoked.

Reply:
Good point, fixed in [3ecc580](https://github.com/jupyrdf/ipyforcegraph/pull/78/commits/3ecc5803d192a278ab2d2e69f38f993244fa9121)